CHANGELOG.md

```diff
@@ -90,6 +90,7 @@
 * Removed deprecated and obsoleted `EventsManager` typealias, `StatusView.delegate`, `StatusView.canChangeValue`, `RouteLegProgress.upComingStep`, `StatusViewDelegate`, `DeprecatedStatusViewDelegate` and `BottomBannerViewController.init(delegate:)`. ([#2993](https://github.com/mapbox/mapbox-navigation-ios/pull/2993))
 * Fixed a potential memory leak when using `MultiplexedSpeechSynthesizer`. ([#3005](https://github.com/mapbox/mapbox-navigation-ios/pull/3005))
 * Fixed an issue where instruction banners could appear in the wrong color after switching between `Style`s. ([#2977](https://github.com/mapbox/mapbox-navigation-ios/pull/2977))
+* Improved the road name labeling accuracy during turn-by-turn navigation by using `Notification.Name.electronicHorizonDidUpdatePosition`. ([#3019](https://github.com/mapbox/mapbox-navigation-ios/pull/3019))
```

Comment:
It‘s sufficient to explain implementation details in this PR’s description; the changelog entry only needs to describe the bug that was fixed.

```suggestion
* Fixed an issue where the current road name label sometimes displayed the name of an intersecting road instead of the current road or blinked in and out. ([#3019](https://github.com/mapbox/mapbox-navigation-ios/pull/3019))
```

Reply:
Thanks for the suggestion, I'll update the ChangeLog.